website/content/docs/job-specification/service.mdx

```diff
@@ -280,6 +291,17 @@ scripts.
 - `tls_skip_verify` `(bool: false)` - Skip verifying TLS certificates for HTTPS
   checks. Requires Consul >= 0.7.2.
 
+- `on_update` `(string: "require_healthy")` - Specifies how the check should be
```

Comment:
Same as above; also, the language doesn't match word-for-word with the above section, which is going to cause a user to pause and try to figure out "Why is this different? Am I missing something?"

Reply:
docs now match word-for-word